Turbulence.ModelGenerator/Converter.cs

```diff
@@ -0,0 +1,500 @@
+                     .Select(x => CultureInfo.InvariantCulture.TextInfo.ToTitleCase(x))
+                     .ToArray()
+                     .Aggregate("", (s1, s2) => s1 + s2.Trim());
+
+        return field.Trim();
+    }
+    
+    private static readonly Dictionary<string, string> DescriptionReplacement = new()
+    {
+        // TODO: Add more replacement strings
+        { "url", "URL" },
+        { "youtube", "YouTube" },
+        { "twitch", "Twitch" },
+        { "oauth2", "OAuth2 "},
+        { "rpc", "RPC" },
+        { " id ", " ID " },
+    };
+
+    private static string PrettifyDescription(string description)
+    {
+        // Convert the first real letter to uppercase
+        char[] chars = description.ToCharArray();
+        for (var i = 0; i < chars.Length; i++)
+        {
+            if (!char.IsLetter(description[i])) continue;
+            
+            chars[i] = char.ToUpper(chars[i]);
+            description = new string(chars);
+            break;
+        }
+
+        description = description[0].ToString().ToUpper() + description[1..];
+
+        if (!description.EndsWith('.')) description += '.';
+
+        // Replace strings in the description to make it prettier
+        description = DescriptionReplacement.Aggregate(description, (curr, map) => 
+                                                           curr.Replace(map.Key, map.Value));
+
+        // Convert markdown hyperlinks
+        const string baseUrl = "https://discord.com/developers/docs";
+        description = Regex.Replace(description, @"\[(.*?)\]\((.*?)\)", @"<a href=""$2"">$1</a>")
+                           .Replace("#DOCS_REFERENCE/", $"{baseUrl}/reference#")
+                           .Replace("#DOCS_RESOURCES_APPLICATION/", $"{baseUrl}/resources/application#")
+                           .Replace("#DOCS_RESOURCES_AUDIT_LOG/", $"{baseUrl}/resources/audit-log#")
+                           .Replace("#DOCS_RESOURCES_CHANNEL/", $"{baseUrl}/resources/channel#")
+                           .Replace("#DOCS_RESOURCES_EMOJI/", $"{baseUrl}/resources/emoji#")
+                           .Replace("#DOCS_RESOURCES_GUILD/", $"{baseUrl}/resources/guild#")
+                           .Replace("#DOCS_RESOURCES_GUILD_SCHEDULED_EVENT/", $"{baseUrl}/resources/guild-template#")
+                           .Replace("#DOCS_RESOURCES_INVITE/", $"{baseUrl}/resources/invite#")
+                           .Replace("#DOCS_RESOURCES_STAGE_INSTANCE/", $"{baseUrl}/resources/stage-instance#")
+                           .Replace("#DOCS_RESOURCES_STICKER/", $"{baseUrl}/resources/sticker#")
+                           .Replace("#DOCS_RESOURCES_USER/", $"{baseUrl}/resources/user#")
+                           .Replace("#DOCS_RESOURCES_VOICE/", $"{baseUrl}/resources/voice#")
+                           .Replace("#DOCS_RESOURCES_WEBHOOK/", $"{baseUrl}/resources/webhook#");
+
+        return description;
+    }
+    
+    public static void PostConvert(Uri tablesPath)
```

Comment:
i presume you wanna add the stuff that is missing here?
at least add some comments.
doing them in code also seems like a big red flag sweetie